app/assets/javascripts/application.js

```diff
@@ -1,3 +1,17 @@
 //= require jquery
 //= require jquery_ujs
 //= require_tree .
+
+$(document).ready(function(){
+  $('.sp_celphones').mask(SPMaskBehavior, spOptions);
```

Comment:
Acho que é melhor deixar o padrão de separar as palavras das classes CSS com hífen (-) 

```suggestion
  $('.sp-celphones').mask(SPMaskBehavior, spOptions);
```